setup.py

```diff
@@ -844,6 +844,13 @@ def configure_extension_build():
             extra_compile_args += ['-g']
             extra_link_args += ['-g']
 
+    # special CUDA 11.7 package that requires installation of cuda runtime, cudnn and cublas
+    pytorch_extra_install_requirements = os.getenv('PYTORCH_EXTRA_INSTALL_REQUIREMENTS', '')
```

Comment:
Nit: use double quotes. Also, why do you need default value here? 
```suggestion
    pytorch_extra_install_requirements = os.getenv("PYTORCH_EXTRA_INSTALL_REQUIREMENTS",)
```

Reply:
Done, converted to double quotes. default values I think good idea to keep in case if environment variable is not set.